Common/AlphaRuntimeStatistics.cs

```diff
@@ -17,34 +20,67 @@ public class AlphaRuntimeStatistics
         /// </summary>
         public AlphaScore RollingAveragedPopulationScore { get; } = new AlphaScore();
 
+        /// <summary>
+        /// Gets the total number of alphas with an up direction
+        /// </summary>
+        public long LongCount { get; set; }
+
+        /// <summary>
+        /// Gets the total number of alphas with a down direction
+        /// </summary>
+        public long ShortCount { get; set; }
+
         /// <summary>
         /// The ratio of <see cref="AlphaDirection.Up"/> over <see cref="AlphaDirection.Down"/>
         /// </summary>
-        public decimal LongShortRatio;
+        public decimal LongShortRatio => ShortCount == 0 ? 1m : LongCount / (decimal) ShortCount;
 
         /// <summary>
         /// The total estimated value of trading all alphas
         /// </summary>
-        public decimal TotalEstimatedAlphaValue;
+        public decimal TotalEstimatedAlphaValue { get; set; }
 
         /// <summary>
         /// The total number of alpha signals generated by the algorithm
         /// </summary>
-        public long TotalAlphasGenerated;
+        public long TotalAlphasGenerated { get; set; }
 
         /// <summary>
         /// The total number of alpha signals generated by the algorithm
         /// </summary>
-        public long TotalAlphasClosed;
+        public long TotalAlphasClosed { get; set; }
 
         /// <summary>
         /// The total number of alpha signals generated by the algorithm
         /// </summary>
-        public long TotalAlphasAnalysisCompleted;
+        public long TotalAlphasAnalysisCompleted { get; set; }
 
         /// <summary>
         /// Gets the mean estimated alpha value
         /// </summary>
         public decimal MeanPopulationEstimatedAlphaValue => TotalAlphasClosed > 0 ? TotalEstimatedAlphaValue / TotalAlphasClosed : 0;
+
+        /// <summary>
+        /// Creates a dictionary containing the statistics
+        /// </summary>
+        public Dictionary<string, string> ToDictionary()
+        {
+            var accountCurrencySymbol = Currencies.GetCurrencySymbol(CashBook.AccountCurrency);
+            return new Dictionary<string, string>
+            {
+                {"Total Alphas Generated", $"{TotalAlphasGenerated}"},
+                {"Total Alphas Closed", $"{TotalAlphasClosed}"},
+                {"Total Alphas Analysis Completed", $"{TotalAlphasAnalysisCompleted}"},
+                {"Long Alpha Count", $"{LongCount}"},
+                {"Short Alpha Count", $"{ShortCount}"},
+                {"Long/Short Ratio", $"{Math.Round(100*LongShortRatio, 2)}%"},
+                {"Total Estimated Alpha Value", $"{accountCurrencySymbol}{TotalEstimatedAlphaValue.SmartRounding()}"},
```

Comment:
Should we tweak this to monthly or this might vary depending on how long the backtest runs?

Reply:
There's another property that is the mean alpha value, basically it's the total/# alphas closed. The normalized value is probably what I would use as a metric over the total value since total value (as mentioned) is proportional to the backtest length